src/v2/api/index.md

```diff
@@ -2197,6 +2197,11 @@ type: api
   ```
 
   `.camel` is not needed if you are using string templates, or compiling with `vue-loader`/`vueify`.
+  
+  ## Truthiness/Falsiness with `v-bind`
```

Comment:
```suggestion
  **Note on truthiness and falsiness
```

Reply:
A level 2 heading here will break the structure of the document/section. I'd suggest using the same style as other list items (Example, See also and so on).

Reply:
Yes, you right